docs/guides/email/02-basic-email-system.md

```diff
@@ -619,37 +668,57 @@ userdb {
 ...
 ```
 
-!!! warning
+!!! warning "Warning"
+
+    Don't write the above grammar on one line, such as this: `userdb {driver = sql args = uid=vmail gid=vmail home=/var/mail/vhosts/%d/%n}`. Otherwise, it won't work.
+
+Edit the `/etc/dovecot/dovecot-sql.conf.ext` file:
 
-    Don't write the above grammar on one line, such as this--"userdb {driver = sql args = uid=vmail gid=vmail home=/var/mail/vhosts/%d/%n}". Otherwise, it won't work.
+```bash
+Shell(192.168.100.7) > vim /etc/dovecot/dovecot-sql.conf.ext
```

Comment:
@jimcat8  and @neelchauhan  

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

Reply:
> @jimcat8 and @neelchauhan
> 
> * what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?

The beginning of the document already indicates that this is an email system composed of three machines.